app/Scripts/UpgradesHandler.gd

```diff
@@ -0,0 +1,19 @@
+extends Node
+
+export(Array, Resource) var presentUpgrades
```

Comment:
what is the purpose of presentUpgrades?

Reply:
It's just an array that holds all the upgrades we already have for further uses, maybe if we have an upgraded upgrade, for example the first one adds 20 to the damage and the second one adds 50 and we need to know whether the player chose the first one or not